### PR TITLE
Improve DS Website Colors page

### PR DESCRIPTION
**Pull request by udimberto**
### What does it do?

Transforming, from images to HTML/selectable content, the Colors page of Strapi DS Website.

### Why is it needed?

Will help the community to select/use/copy the Strapi DS color tokens more easier.

### CODE REVIEW

**Comment by Eventyret** [comment]
LGTM @udimberto 🥳

**Review by joshuaellis — DISMISSED** [review]
Nice changes all around @udimberto <3

**Review by gu-stav — COMMENTED** [review]
That looks very good indeed! Thanks!


What do you think about improving the accessibility of these cards before we merge it? Right now there are only `div`s which makes them very much not accessible.

Screenshot 2023-04-11 at 12 53 05

Markup-wise I could think of:

- each card is an `article`
- each `article` has a headline with the name of the color (hidden)
- all other key-value pairs are a `dl/dt` structure
- the font-size icons could get a hidden aria label indicating "Large font" / "Small font" for example and be a `dl/dt` too?

**Review by HichamELBSI — APPROVED** [review]

**Comment by udimberto** [comment]
> 

I think that's great, @gu-stav 😄.

Awesome suggestions, thank you so much.
I've already applied them (_not very sure about what I did_ 😅).

Would you mind to give a second look, when you have a free time, please?

**Comment by udimberto** [comment]
> That looks very good indeed! Thanks!
> 
> What do you think about improving the accessibility of these cards before we merge it? Right now there are only `div`s which makes them very much not accessible.
> 
> Screenshot 2023-04-11 at 12 53 05
> 
> Markup-wise I could think of:
> 
> * each card is an `article`
> * each `article` has a headline with the name of the color (hidden)
> * all other key-value pairs are a `dl/dt` structure
> * the font-size icons could get a hidden aria label indicating "Large font" / "Small font" for example and be a `dl/dt` too?

I think that's great, @gu-stav 😄.

Awesome suggestions, thank you so much.
I've already applied them (_not very sure if I've applied them correctly_ 😅).

Would you mind to give a second look, when you have a free time, please?

File: website/components/MarkdownComponents/ColorShades.jsx
Comment (joshuaellis) [inline website/components/MarkdownComponents/ColorShades.jsx:29]:
```suggestion
        as="dt"
```
?

**Review by joshuaellis — CHANGES_REQUESTED** [review]

File: website/components/MarkdownComponents/ColorShades.jsx
Comment (joshuaellis) [inline website/components/MarkdownComponents/ColorShades.jsx:88]: I think because you're not doing a grid here you're just doing a row, we should use `Flex`, it'll reduce the HTML too and get rid of `GridItem` 👍🏼

Comment (udimberto) [reply website/components/MarkdownComponents/ColorShades.jsx]: Yes, it worked.
Changes applied.
Thank you for that.
🤝

**Review by gu-stav — APPROVED** [review]
Great work, @udimberto. I think this is a very good start!

I've tried using the page using voiceover today and could think of a few improvements, but would leave it up to you if you want to implement those:

- The colors at the top are focusable (probably because they have a tooltip). I was wondering if it would make sense to link them to the headlines below (with an anchor) or remove them at all? I don't see how they improve the information on the page.

Screenshot 2023-04-14 at 08 42 52

- You could wrap each `article` in an ordered-list item, since the color shades are a list of things. This would allow screen-reader users more easily to discover how many shades each color contains and to skip over them. I don't think it is major because the page is already grouped by headlines and screenreader users could skip to the next one.

- The labels "FAIL" and "PASS" are currently written in uppercase in the markup. I think it would be better to use `text-transform: uppercase` for it.

Again: good work! 🚀

**Review by joshuaellis — APPROVED** [review]

**Review by Bassel17 — APPROVED** [review]
ohhh that's a great improvement love this !

**Pull request merged**
